autogtp/Management.cpp

```diff
@@ -472,17 +472,26 @@ Order Management::getWork(bool tuning) {
 bool Management::networkExists(const QString &name) {
     QString realHash = name;
     realHash.remove(0,9);
+    realHash.chop(3);
     if (QFileInfo::exists(name)) {
-        QFile f(name);
+#ifdef WIN32
+        QProcess::execute("gzip.exe -k -d -q " + name);
```

Comment:
Dawww :(

Reply:
Is it necessary to check that the uncompressed network matches? It isn't really for security as the client trusts the server to provide the right file. A server calculated gziphash is still trusting the server. And even before, the server could have just told the client to download some other "network" that had the appropriate hash in the filename.

Reply:
But even with those types of network / file failures, it would likely result in leelaz complaining "Weights file is the wrong version."

Reply:
File size check may be risky as all files are approximately the same size.

>it would likely result in leelaz complaining "Weights file is the wrong version." or "Failed to parse weight file. Error on line 2."

Yes, but AutoGTP won't know the underlying reason then.